src/timesync/timesyncd-manager.c

```diff
@@ -378,10 +378,10 @@ static int manager_adjust_clock(Manager *m, double offset, int leap_sec) {
         m->drift_ppm = tmx.freq / 65536;
 
         log_debug("  status       : %04i %s\n"
-                  "  time now     : %li.%03"PRI_USEC"\n"
-                  "  constant     : %li\n"
+                  "  time now     : %"PRI_TIMEX".%03"PRI_USEC"\n"
```

Comment:
`tmx.time.tv_sec` is of type `time_t`. It should be printed using `PRI_TIME`, not `PRI_TIMEX`.